torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -423,6 +423,21 @@ def is_zero_dim(meta):
             # TODO: no ttnn op can convert _adaptive_avg_pool2d
             return self.call_function_prop_meta(target, args, kwargs)
 
+        if target == torch.ops.aten._scaled_dot_product_flash_attention.default:
+
+            def select(dropout_p=0.0, is_causal=False):
+                # TODO(jdh8): Add suuport for training mode
+                if dropout_p > 0.0:
+                    return self.call_function_prop_meta(target, args, kwargs)
+
+                return self.call_function_prop_meta(
+                    ttnn.transformer.scaled_dot_product_attention,
+                    args[:3],
+                    {"is_causal": is_causal},
+                )
+
+            return select(*args[3:])
```

Comment:
need to fire a ticket for unsupported cases?

Reply:
I'm marking this issue as a feature request tenstorrent/tt-metal#16022.  No input variation now has nonzero `dropout_p` yet.  It's still good to keep an eye.

Reply:
Doesn't this logic drop the attention mask, which must be provided if `is_causal == False`

Reply:
[`aten._scaled_dot_product_flash_attention`](https://pytorch.org/cppdocs/api/function_namespaceat_1a99894de13fe7feccc3bcf084ee870f02.html) does not provide attention mask as far as I know.  I have not yet found better documentation.  Please correct me if I am wrong.